docs/source/architecture/02_validated_architecture.md

```diff
@@ -0,0 +1,52 @@
+# Validated Architecture
```

Comment:
Here we use the term "validated architectures" and present a virtualized environment. This is unlikely given that we want to test VAs on baremetal setups. Is this just a representation of how we will be doing this right now?

Reply:
It's deploying a virtual layout matching as closely as possible what we expect to get in real life. This is, really, CI/Dev oriented, so we have to work with what we have.
Also, hybrid (virtual OCP + actual baremetal computes) should be supported with little effort I think.

Reply:
Do we want to tie VA to baremetal? I personally think the underlying hardware should not be part of the VA definition, but just an additional testing detail. In this sense this section can be allowed to stay, maybe some additional wording ("While a full testing of a VA should be done on baremetal,... blablalba")